examples/src/main/scala/com/example/http4s/ScienceExperiments.scala

```diff
@@ -103,21 +102,28 @@ object ScienceExperiments {
 
     ///////////////// Weird Route Failures //////////////////////
     case req @ GET -> Root / "hanging-body" =>
-      Ok(Process(Task.now(ByteVector(Seq(' '.toByte))), Task.async[ByteVector] { cb => /* hang */}).eval)
+      Ok(Stream.eval(Task.now(ByteVector(Seq(' '.toByte))))
+        .evalMap[Task, Task, Byte](_ => Task.async[Byte]{ cb => /* hang */}))
 
     case req @ GET -> Root / "broken-body" =>
-      Ok(Process(Task{"Hello "}) ++ Process(Task{sys.error("Boom!")}) ++ Process(Task{"world!"}))
+      Ok(Stream.eval(Task{"Hello "}) ++ Stream.eval(Task{sys.error("Boom!")}) ++ Stream.eval(Task{"world!"}))
 
-    case req @ GET -> Root / "slow-body" =>
-      val resp = "Hello world!".map(_.toString())
-      val body = awakeEvery(2.seconds).zipWith(Process.emitAll(resp))((_, c) => c)
-      Ok(body)
+//    Missing Strategy in Scope Which Is Clearly Not the Case
+//    case req @ GET -> Root / "slow-body" =>
+//      val resp = "Hello world!".map(_.toString())
+//      val body = time.awakeEvery(2.seconds).zipWith(Stream.emits(resp))((_, c) => c)
+//      Ok(body)
 
     case req @ POST -> Root / "ill-advised-echo" =>
       // Reads concurrently from the input.  Don't do this at home.
-      implicit val byteVectorMonoidInstance: Monoid[ByteVector] = Monoid.instance(_ ++ _, ByteVector.empty)
-      val tasks = (1 to Runtime.getRuntime.availableProcessors).map(_ => req.body.foldMonoid.runLastOr(ByteVector.empty))
-      val result = Task.reduceUnordered(tasks)(Reducer.identityReducer)
+      implicit val byteVectorMonoidInstance: Monoid[ByteVector] = new Monoid[ByteVector]{
```

Comment:
We had this monoid in 0.16, and removed it when ByteVector was temporarily removed from the classpath in 0.17.  It might be worth bringing back.